dashboard/app/models/levels/external.rb

```diff
@@ -52,6 +55,10 @@ def concept_level?
     true
   end
 
+  def display_as_unplugged
+    properties.merge({'display_as_unplugged' => !properties['display_as_unplugged']})
```

Comment:
I'm confused by this.  Why does this method return a hash? And why does it invert the `display_as_unplugged` value?

Reply:
oh good call.  The form field is doing more for me than I thought. removed!